addons/website_sale_wishlist/static/src/js/website_sale_tour_wishlist.js

```diff
@@ -0,0 +1,118 @@
+odoo.define('website_sale_wishlist.tour', function (require) {
+'use strict';
+
+var tour = require("web_tour.tour");
+var base = require("web_editor.base");
+
+tour.register('shop_wishlist', {
+    test: true,
+    url: '/shop',
+    wait_for: base.ready(),
+},
+    [
+        {
+            content: "click on add to wishlist",
+            trigger: '.o_add_wishlist',
+        },
+        {
+            content: "go to wishlist",
+            extra_trigger: ".label:contains(1)",
+            trigger: 'a[href="/shop/wishlist"]',
+        },
+        {
+            content: "remove Ipad",
+            trigger: 'a.o_wish_rm:first',
+        },
```

Comment:
check that wishlist is now in cart
Cart:contains 1
Wishlist: contains: 0

check that returned / view empty wishlist msg